src/tardev-snapshotter/tarindex/src/lib.rs

```diff
@@ -306,7 +301,7 @@ pub fn append_index(data: &mut (impl io::Read + io::Write + io::Seek)) -> io::Re
         mode: S_IFDIR | 0o555,
         ..Entry::default()
     }));
-
+    let mut special_link: Vec<Vec<u8>> = Vec::new();
```

Comment:
nit: I believe the type can be inferred here, so it can probably be omitted.

Reply:
Yes. type omitted.